src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -478,14 +540,18 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             bool disableNotification = default,
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
+            CancellationToken cancellationToken = default,
+            bool allowSendingWithoutReply = default,
+            IEnumerable<MessageEntity> captionEntities = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.